bcda/client/bluebutton_test.go

```diff
@@ -871,10 +875,7 @@ func (s *BBRequestTestSuite) TestValidateRequest() {
 		s.T().Run(tt.name, func(t *testing.T) {
 
 			tsValidation := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-				uid, err := uuid.Parse(req.Header.Get("BlueButton-OriginalQueryId"))
```

Comment:
It looked like we had two different UUID packages we used in the repo. updated to the more commonly used one here, which changes the return value of `Parse`, hence the change.